### PR TITLE
Booklet Page Polish

### PR DESCRIPTION
**Pull request by jayjhan8**

### CODE REVIEW

**Review by tombak98 — APPROVED** [review]

**Comment by younghj** [comment]
hey jay, it seems like the responsive effect only happens once i click onto the next image.
and the pages for the mobile (the 6th to 9th pages), the sizes change and the arrows beside the images move up and down.

can you check if this is a issue for me or you see this too?

**Comment by younghj** [comment]
for the responsive thing, maybe using this as reference can help
https://github.com/wliu080/japan-prayer-guide/blob/03fc197933fd5b235f49101aabdbe375ddc0839a/components/toggleHeader.tsx#L45C3-L67C66

File: pages/booklet.tsx [hunk 1]
```diff
@@ -1,39 +1,45 @@
 import Head from "next/head"
 import Link from "next/link"
-import React from "react"
+import React, {useEffect} from "react"
```
Comment (younghj) [inline pages/booklet.tsx:3]: oh i see you added useEffect here, i think using this with this code as reference could help!
https://github.com/wliu080/japan-prayer-guide/blob/03fc197933fd5b235f49101aabdbe375ddc0839a/components/toggleHeader.tsx#L45C3-L67C66

**Review by younghj — CHANGES_REQUESTED** [review]

File: pages/booklet.tsx (hunk 1 above)
Comment (jayjhan8) [reply pages/booklet.tsx:3]: I think the `react-responsiveness` is cleaner, but we could probably discuss this after the conference.  This `useEffect` was left from my testing with a previous version so I need to remove it

**Comment by jayjhan8** [comment]
@younghj I've fixed the EN mobile image sizes so you shouldn't see the arrows move.

I'm not sure if I know what you mean by "responsive effect only happens once i click onto the next image."   

I did notice that there is a bug if you were to navigate to the 6th image in mobile and resize your screen up to desktop, this breaks the indexing because there are only 5 images on desktop vs 9 on mobile.  I'll follow up with a bug fix, but I'm not too concerned with this for the conference.

**Comment by younghj** [comment]
oh its interesting, the responsive thing is fixed now, but the sizing thing still remains..

https://github.com/wliu080/japan-prayer-guide/assets/7674275/ad0d9d05-9de2-4705-9fdb-e4c5f4ad6134

**Review by younghj — APPROVED** [review]

**Pull request merged**